apps/auth/models.py

```diff
@@ -31,3 +32,16 @@ def __init__(self, username: str, email: str, password=None):
 
     def set_password(self, password):
         self.password = generate_password_hash(password)
+
+    def get_last_10_quiz_attempts(self):
+        this_user_attempts = UserQuizAttempts.query.filter_by(user_id=self.id).order_by(UserQuizAttempts.date.desc()).limit(10).all()
+        if this_user_attempts == []:
+            return "You don't have any quiz attempts yet"
```

Comment:
This should return None/empty query

Reply:
But it doesn't, I tested it on an empty table

Reply:
What I mean is that model method should not return a message, it should return either None, or empty query, message should be handled in views

Reply:
My bad, I misunderstood, I'll change it later today